index.html

```diff
@@ -11,14 +11,79 @@
     />
   </head>
   <body class="app-grid">
-    <section class="header"></section>
 
-    <section class="content"></section>
+    <section class="header header__bar">
+      <p>Quiz App</p>
```

Comment:
This is a title so use a headline element instead.
```suggestion
      <h1>Quiz App</h1>
```